languages/ruby/lib/ruby/sdk/version.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Ruby
+  module Bitwarden
+    module Sdk
+      VERSION = '0.1.8'
```

Comment:
Can we change this to 0.0.0, we'll want the CI job to burn in the actual version.